File: core/src/com/qtech/bubbleblaster/command/BloodMoonCommand.java
package com.qtech.bubbleblaster.command;

import com.qtech.bubbleblaster.common.command.CommandExecutor;
import com.qtech.bubbleblaster.entity.player.PlayerEntity;

public class BloodMoonCommand implements CommandExecutor {
    @Override
    public boolean execute(PlayerEntity player, String[] args) {
        player.getGameType().triggerBloodMoon();
        return true;
    }
}

File: src/main/kotlin/qtech/bubbles/command/BloodMoonCommand.kt
package qtech.bubbles.command

import qtech.bubbles.common.command.CommandExecutor
import qtech.bubbles.entity.player.PlayerEntity

open class BloodMoonCommand : CommandExecutor {
    override fun execute(player: PlayerEntity, args: Array<String>): Boolean {
        player.gameMode.triggerBloodMoon()
        return true
    }
}